notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -12041,6 +12041,51 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
 incorporeal being [Spencer's body had been taken over by a malevolent alien entity that, at least according to Sherman Reese, was traveling space as a transmission at the speed of light.]
 
 
+movie: Peutetre (1999)
+======================
+
+:: Title
+Peut-être
+
+:: Description
+Peut-être (Maybe; Perhaps) is a 1999 French science fiction comedy film. After
+having sex with his girlfriend Lucie in a bathroom, Arthur discovers that a
+ceiling panel is a time portal to the Paris in the future, although it appears
+more like a sun-baked desert city by that point. There he meets an old man
+named Ako who turns out to be Arthur's son. Ako attempts to persuade Arthur to
+impregnate Lucie so that he can exist in this future.
+
+:: Date
+1999-11-10
+
+:: References
+https://en.wikipedia.org/wiki/Peut-%C3%AAtre
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I met my distance descendants [The main novelty of the film concerns the young man Arthur meeting his future children and grandchildren after passing through a time portal.]
+father and son [The young man Arthur traveled into the future whereupon he met his future son, Ako, who was by then an old man. The story then follows Ako's attempts to persuade Arthur to impregnate his girlfriend so that Ako and his progeny can continue to exist in the future.]
+
+:: Major Themes
+what if I found myself in the future [Arthur passed through a time portal to find himself 60 years in the future. Not only did he find the Paris of the future to be a sun-baked desert, but he also met his future son and grandchildren.]
+personal practical dilemma [Should Arthur impregnate his girlfriend or not? If he did, then his future son Ako and Ako's descendants would go on living happily in the future; otherwise they would cease to exist.]
```

Comment:
I don't quite see how "time traveler's dilemma" applies here. Care to elaborate?

That aside, the main character was grappling with some or another though decision. I suppose we could just use the general "personal life and death dilemma".

Reply:
There is a variant of the grandfather paradox (which we reference in our theme def). Ako didn't go back in time to kill his father, but to convince his father to impregnate his mother. That's stupid of course since it means a non-existing person brought about his own existence. A paradox. But such is the story, if I read your comments correctly.

Reply:
I could have swore that I replied here, but I guess not. In any case, I added **time traveler's dilemma**.